index.html

```diff
@@ -164,14 +164,15 @@ <h2 class="contact-header">Contact me</h2>
 				it!
 			</p>
 			<div class="contact-form">
-				<form action="https://formspree.io/f/mlekpene" method="post">
+				<form action="https://formspree.io/f/mlekpene" method="post" class="form">
             <input
               type="text"
               maxlength="30"
               placeholder="Lorem Ipsum"
               required
               name="name" />
             <input
+              class="email"
               type="email"
               name="email"
               placeholder="loremipsum@gmail.com"
```

Comment:
- [ ] [OPTIONAL]
I think it will be a good idea to improve the design of your form on desktop version to match the chosen figama template.

**Your form [Desktop view]:**
![image](https://user-images.githubusercontent.com/12745474/231564190-a75d9d53-9692-4e6d-94b7-f4c078b51032.png)

Reply:
thank you to inform me